minecraft_servers/reberion/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "reberion",
+  "nice_name": "Reberion Network",
+  "direct_ip": "reberion.eu",
+  "server_wildcards": [
+    "%.reberion.eu"
+  ],
+  "social": {
+    "web": "https://www.reberion.eu",
+    "twitter": "ReberionFeeds",
+    "teamspeak": "ts.reberion.eu",
+    "discord": "https://reberion.eu/discord",
+    "web_shop": "https://shop.reberion.eu",
+    "web_support": "https://support.reberion.eu"
+  },
+  "discord": {
+    "server_id": 1018169330875236362,
+    "rename_to_minecraft_name": true

```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.